float64[] r_eigs
float64[] q_eigs
---
float64[] results
